pkg/master/master.go

```diff
@@ -354,14 +355,18 @@ type nodeAddressProvider struct {
 }
 
 func (n nodeAddressProvider) externalAddresses() (addresses []string, err error) {
+	preferredAddressTypes := []api.NodeAddressType{
```

Comment:
I misunderstood the tunneling healthz. The tunneling shouldn't use the InternalIP.

Now the changes to this file do not change any logic. The only change is reusing the utility function in pkg/util/node.